build/configs/artik053/extra/defconfig

```diff
@@ -1139,6 +1138,12 @@ CONFIG_TASH_CMDTASK_PRIORITY=100
 # CONFIG_SYSTEM_FOTA_HAL is not set
 # CONFIG_SYSTEM_I2CTOOL is not set
 # CONFIG_SYSTEM_INIFILE is not set
+CONFIG_SYSTEM_PREAPP_INIT=y
+CONFIG_SYSTEM_PREAPP_STACKSIZE=2048
+
+#
+# FLASH Program Installation
```

Comment:
In apps/system/init/Kconfig file.
config SYSTEM_PREAPP_INIT
    bool "Preapp Init"
    default y
    ---help---
        Enable Preapp Init.

if SYSTEM_PREAPP_INIT

config SYSTEM_PREAPP_STACKSIZE
    int "pre-app thread stack size"
    default 2048
    ---help---
        The size of the stack to allocate for the pre-application thread
        that is started as soon as the OS completes its initialization.

endif